tests/templates/DonationForm.html.twig

```diff
@@ -0,0 +1,18 @@
+Amount: {$ betrag $}
+Payment type: {$ zahlweise $}
+Address type: {$ adresstyp $}
+Interval: {$ periode $}
+IBAN: {$ iban $}
+BIC: {$ bic $}
+Bank name: {$ bankName $}
+Address type: {$ adressType $}
+Salutation: {$ anrede $}
+Title: {$ titel $}
+Company: {$ firma $}
+First name: {$ vorname $}
+Last name: {$ nachname $}
+Street: {$ strasse $}
+Postal code: {$ plz $}
+City: {$ ort $}
+Country code: {$ country $}
+Email address: {$ email $}
```

Comment:
So this is a placeholder form, that should still be replaced by some real frontend stuffs? How would that work?